packages/cli/package.json

```diff
@@ -99,6 +99,7 @@
     "@types/got": "^9.6.12",
     "@types/inquirer": "^9.0.3",
     "@types/lodash": "^4.14.192",
-    "@types/yargs": "^17.0.24"
+    "@types/yargs": "^17.0.24",
+    "@lodestar/test-util": "^1.9.1"
```

Comment:
if we want the dependencies in alpha order, this should come first

Reply:
related https://github.com/ChainSafe/lodestar/pull/5648#discussion_r1250682075, maybe should introduce something like [sort-package-json](https://www.npmjs.com/package/sort-package-json) as a CI lint job, unordered deps caused merge issues in the past

Reply:
I used the VSCode feature auto-sort lines.